### PR TITLE
Issue #341: Add missing components and update existing ones

### LINKED ISSUE #341: Add missing components.
There are a few component missing.

### PR DESCRIPTION
**Pull request by drupol**
Fix #341

### CODE REVIEW

File: atomium/templates/exposed_filters/exposed_filters.component.inc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    _atomium_extend_theme_property(
+      $exposed_filters,
+      array(
+        $variables['theme_hook_original'],
+      )
+    );
+  }
+
+  $variables[$variables['base_hook']] = array(
```
Comment (aritomelo) [inline atomium/templates/exposed_filters/exposed_filters.component.inc:53]: Why this part is needed?

Comment (drupol) [reply atomium/templates/exposed_filters/exposed_filters.component.inc:53]: Before rendering the template, Drupal extract the `$variables` and as we uses `$exposed_filters` in the template, we need to do that.

See: https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme_render_template/7.x

File: atomium/templates/pager_last/pager_last.component.inc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+function atomium_preprocess_pager_last(array &$variables, $hook) {
+  $text = $variables['text'];
+  $element = $variables['element'];
+  $parameters = $variables['parameters'];
+  global $pager_page_array, $pager_total;
+  $render = NULL;
+
+  $pager_last = NULL;
+
+  // If we are anywhere but the last page.
```
Comment (aritomelo) [inline atomium/templates/pager_last/pager_last.component.inc:54]: What does mean the comment?

File: atomium/templates/system_powered_by/system-powered-by.tpl.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * @file
+ * Contains template file.
+ */
+?>
+<span>
```
Comment (aritomelo) [inline atomium/templates/system_powered_by/system-powered-by.tpl.php:8]: @drupol I would suggest using **t** function in preprocessing and on template just print the vars, this way all the logic can be treated in preprocess and also easy future changes, avoiding touching the template file as much as possible.
What do you think?
My suggestion would be like [this](https://3v4l.org/rMMJ3)

Comment (drupol) [reply atomium/templates/system_powered_by/system-powered-by.tpl.php:8]: Ok !

Then I would use:

```php
$variables[$variables['base_hook']] = t('Powered by <a href="@poweredby">Drupal</a> & <a href="@atomium_url">Atomium</a>', array('@poweredby' => 'https://www.drupal.org', '@atomium_url' => 'https://drupal.org/project/atomium'))
```

File: atomium/templates/exposed_filters/exposed_filters.component.inc (hunk 1 above)
Comment (aritomelo) [reply atomium/templates/exposed_filters/exposed_filters.component.inc:53]: Cheers!

File: atomium/templates/pager_last/pager_last.component.inc (hunk 2 above)
Comment (drupol) [reply atomium/templates/pager_last/pager_last.component.inc:54]: If we are on any element except the last page element of the pager.

**Review by aritomelo — APPROVED** [review]
CR has done and ok to be merged.

File: atomium/templates/system_powered_by/system-powered-by.tpl.php (hunk 3 above)
Comment (drupol) [reply atomium/templates/system_powered_by/system-powered-by.tpl.php:8]: Fixed in #344

**Pull request merged**